homeassistant/components/radarr/sensor.py

```diff
@@ -1,266 +1,185 @@
 """Support for Radarr."""
 from __future__ import annotations
 
-from datetime import datetime, timedelta
-from http import HTTPStatus
-import logging
-import time
-from typing import Any
+from collections.abc import Callable
+from copy import deepcopy
+from dataclasses import dataclass
+from typing import Generic
 
-import requests
+from aiopyarr import RootFolder
 import voluptuous as vol
 
 from homeassistant.components.sensor import (
     PLATFORM_SCHEMA,
     SensorEntity,
     SensorEntityDescription,
 )
+from homeassistant.config_entries import SOURCE_IMPORT, ConfigEntry
 from homeassistant.const import (
     CONF_API_KEY,
     CONF_HOST,
     CONF_MONITORED_CONDITIONS,
     CONF_PORT,
     CONF_SSL,
     DATA_BYTES,
-    DATA_EXABYTES,
     DATA_GIGABYTES,
     DATA_KILOBYTES,
     DATA_MEGABYTES,
-    DATA_PETABYTES,
-    DATA_TERABYTES,
-    DATA_YOTTABYTES,
-    DATA_ZETTABYTES,
 )
 from homeassistant.core import HomeAssistant
 import homeassistant.helpers.config_validation as cv
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
-from homeassistant.helpers.typing import ConfigType, DiscoveryInfoType
-from homeassistant.util import dt as dt_util
+from homeassistant.helpers.typing import ConfigType, DiscoveryInfoType, StateType
 
-_LOGGER = logging.getLogger(__name__)
+from . import RadarrEntity
+from .const import DEFAULT_NAME, DOMAIN
+from .coordinator import RadarrDataUpdateCoordinator, T
 
-CONF_DAYS = "days"
-CONF_INCLUDED = "include_paths"
-CONF_UNIT = "unit"
-CONF_URLBASE = "urlbase"
 
-DEFAULT_HOST = "localhost"
-DEFAULT_PORT = 7878
-DEFAULT_URLBASE = ""
-DEFAULT_DAYS = "1"
-DEFAULT_UNIT = DATA_GIGABYTES
+def get_space(coordinator: RadarrDataUpdateCoordinator, name: str) -> str:
+    """Get space."""
+    space = [
+        mount.freeSpace / 1024 ** BYTE_SIZES.index(DATA_GIGABYTES)
+        for mount in coordinator.data
+        if name in mount.path
+    ]
+    return f"{space[0]:.2f}"
+
+
+def get_modified_description(
```

Comment:
This isn't readable. Just append the name to the entity name and unique_id in the entity init method instead. The description can have a function that returns the name suffix to be appended. Don't modify the description.